### DISCUSSION TITLE
provide UI/UX prototypes

### DISCUSSION THREAD

**Discussion by struckchure** [Dev-47/quiz-app#11] [General]
Either gather them from different sources into one or make the them from scratch or existing resources

**Comment by struckchure** [comment]
This needs to be submitted latest on Sunday, hopefully @brafii can meet up

And we should start looking for prototypes from Figma and anywhere else we can find go ones

**Comment by struckchure** [comment]
Attention is needed !!!!!!

**Comment by struckchure** [comment]
**Pages needed**

- login / register / forgot password
- profile page
- dashboard
  - to display quiz created by the user (list of quizzes created by user)
  - and the number of quiz answers (plus short description)
- page to add a new quiz (add a kind of switch or button to activate -ve marking)
  - quiz questions with options
- page to answer a quiz and get result result

** details **

- page to add new quiz should contain;
  - quiz title, button to dynamically add questions, initially should contain one component
  - the question component should have a delete button too
  - question component would have button to dynamically add options

I'll add more as it comes

**Reply by brafii** [reply]
Alright seen

**Comment by Haryanfe** [comment]
Okay, but still don't know how to help

**Reply by brafii** [reply]
message me on WhatsApp lets discuss search for @brafii on the group

**Comment by struckchure** [comment]
@Dev-47/fe 

You all need to checkout the prototype for the mobile version

https://xd.adobe.com/view/70264988-c865-4fb2-94dd-0e82403d5e8b-2575/

**Reply by horlakz** [reply]
Can this Mobile version be used used for the web mobile version?

**Reply by struckchure** [reply]
Yes, most part of it
Like the authentication pages

**Comment by struckchure** [comment]
@Dev-47/fe 

The web prototype is ready

https://xd.adobe.com/view/eff255b1-4d02-41cc-8159-0e941aa3ab29-baa1/

**Reply by horlakz** [reply]
alright

**Comment by struckchure** [comment]
[Assets from protype](https://github.com/Dev-47/quiz-app/files/7784991/images.zip)

@Dev-47/fe
